Makefile.am

```diff
@@ -535,7 +535,7 @@ DISTCHECK_CONFIGURE_FLAGS+= --enable-mmtaghostname
 endif
 
 if ENABLE_IMTUXEDOULOG
-DISTCHECK_CONFIGURE_FLAGS+= --enable-imtuxedolog
+DISTCHECK_CONFIGURE_FLAGS+= --enable-imtuxedoulog
```

Comment:
This cannot be changed, it breaks backward copatibility - plus it makes little sense, as this is the name of a component.

Reply:
Sorry, the correct fix is: https://github.com/jsoref/rsyslog/commit/6bc5a94d64a0a10248a13156a1e354421a44b787

What I'm doing is looking at two tokens and picking one, I picked the wrong one.